exes/mqtt-broadcaster/CMakeLists.txt

```diff
@@ -1,16 +1,21 @@
-add_executable(mqtt-broadcaster
+add_executable(mqtt_broadcaster
+  src/impl.cpp
```

Comment:
impl of what?

Reply:
These are shared utility functions that mutliple classes use, change name to utility.cpp / utility.hpp?

Reply:
maybe `sparkplug_utils` ? `utility` does not say its purpose other than being a blackhole of random stuff

Reply:
changed name to `sparkplug_utils`